.gitignore

```
@@ -194,3 +194,6 @@ cython_debug/
 
 # TorchScript model
 *.pt
+
+# Apple Desktop Services Store
+*.DS_Store
```

IIRC this will ignore anything ending in .DS_Store in the top level directory, but `**` will ignore in every directory. see https://git-scm.com/docs/gitignore#_pattern_format

```suggestion
**.DS_Store
```